javers-core/src/main/java/org/javers/common/exception/JaversExceptionCode.java

```diff
@@ -40,6 +40,10 @@
 
     SET_OF_VO_DIFF_NOT_IMPLEMENTED("diff for Set of ValueObjects is not supported"),
 
+    MULTISET_OF_VO_DIFF_NOT_IMPLEMENTED("diff for Multiset of ValueObjects is not supported"),
```

Comment:
już jest :) tzn w masterze jest już wsparcie dla Setów VO, jeśli zrobisz update z mastera to będziesz mógł też zaaplikować wsparcie dla VO po stronie Multisetów